src/gradescopeapi/_classes/_account.py

```diff
@@ -70,3 +77,124 @@ def get_courses(self) -> dict:
         courses["student"] = student_courses
 
         return courses
+
+    def get_assignments(self, course_id: int) -> List[Dict[str, str]]:
```

Comment:
```suggestion
    def get_assignments(self, course_id: str) -> List[Dict[str, str]]:
```
I think we are keeping `course_id` as strings for now. I think keeping them as `ints` would also be OK but we would need to change all places we are using the `course_id` to `int`